src/test/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentServiceTest.kt

```diff
@@ -74,6 +77,29 @@ class TaskDeploymentServiceTest {
             val alias = it.getArgument<String>(0).split("/").last()
             "arn:aws:kms:${configurationResolver.awsRegion}:000:key/testkeyarn-$alias"
         }
```

Comment:
there should be a test for the null return that is hinted at in if statement line 342 in TaskDeploymentService (`if (response != null) { ...`)